cim-forward-route/src/main/java/com/crossoverjie/cim/route/config/BeanConfig.java

```diff
@@ -109,6 +114,27 @@ public RouteHandle buildRouteHandle() throws Exception {
 
     }
 
+    @Bean
```

Comment:
```suggestion
    @Bean("userInfoCache")
```

Reply:
There may be multiple instances of the base cache, and it is recommended to use names to distinguish them.